client/js/components/statement/assessmentTable/DpAssessmentTableCard.vue

```diff
@@ -851,7 +868,8 @@ export default {
     },
 
     publicVerifiedKeyIcon () {
-      let icon = 'fa-eye-slash color--grey'
+      let icon
```

Comment:
sonar cloud complained about that. `'fa-eye-slash color--grey'` is already declaired als default in the switch...